src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+    >
+
+      <img
+        src="images/Search.svg"
+        alt="big serch"
```

Comment:
the `alt` has typo in word `serch` -> `search`
```suggestion
        alt="big search"
```

Reply:
and check in other namings please